api/types.go

```diff
@@ -0,0 +1,47 @@
+package api
+
+type Context struct {
+	State DebuggerState `json:"state"`
```

Comment:
@derekparker The pattern's intent is basically to separate in an object which contains both request information and status, which is what.  In this case I'm thinking something like:

```
DebuggerContext {
  Requested: {
    State: Continue
  },
  Status: {
    State: Paused,
    Thread: foo,
    Func: doSomething,
    Line: 123,
  }
}